cmake/projects/Fruit/hunter.cmake

```diff
@@ -0,0 +1,32 @@
+# cmake/projects/Fruit/hunter.cmake
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+	PACKAGE_NAME
+	Fruit
+	VERSION
+	"3.1.1"
```

Comment:
It's better to have the version and tag match together, please use `3.1.1-p0` here.